missing definition of USBD_LL_Init_Done

when building libcanokey-qemu.so one gets a library with one missing symbol:

```
$  nm -DC /nix/store/w0lh3v93bjid2w4z4ngps4q6qd7lxrx4-canokey-qemu-0-unstable-2026-03-15/lib/libcanokey-qemu.so | grep USBD_LL_
0000000000020b40 T USBD_LL_ClearStallEP
0000000000020b20 T USBD_LL_CloseEP
0000000000031740 T USBD_LL_DataInStage
0000000000031680 T USBD_LL_DataOutStage
0000000000020ac0 T USBD_LL_DeInit
0000000000031930 T USBD_LL_DevConnected
0000000000031940 T USBD_LL_DevDisconnected
0000000000020b30 T USBD_LL_FlushEP
0000000000020c00 T USBD_LL_GetRxDataSize
0000000000020ab0 T USBD_LL_Init
                 U USBD_LL_Init_Done
0000000000020b50 T USBD_LL_IsStallEP
0000000000020b10 T USBD_LL_OpenEP
0000000000020ba0 T USBD_LL_PrepareReceive
0000000000031860 T USBD_LL_Reset
0000000000031900 T USBD_LL_Resume
00000000000318d0 T USBD_LL_SetSpeed
0000000000031590 T USBD_LL_SetupStage
0000000000020b60 T USBD_LL_SetUSBAddress
0000000000031920 T USBD_LL_SOF
0000000000020b80 T USBD_LL_StallEP
0000000000020ad0 T USBD_LL_Start
0000000000020b00 T USBD_LL_Stop
00000000000318e0 T USBD_LL_Suspend
0000000000020bd0 T USBD_LL_Transmit
```
Notice how `USBD_LL_Init_Done` is missing

Unsurprisingly when building qemu one gets a linker error:
```
qemu> [7028/7341] Linking target qemu-system-mips
qemu> FAILED: [code=1] qemu-system-mips 
qemu> gcc -m64 @qemu-system-mips.rsp
qemu> /nix/store/9nmzd62x45ayp4vmswvn6z45h6bzrsla-binutils-2.44/bin/ld: /nix/store/w0lh3v93bjid2w4z4ngps4q6qd7lxrx4-canokey-qemu-0-unstable-2026-03-15/lib/libcanokey-qemu.so: undefined reference to `USBD_LL_Init_Done'
```

affected commit: current master c88528d5cf756248c26189145d601bac7fccb104

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,6 +10,7 @@ set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -fno-omit-frame-pointer")
 option(QEMU_DEBUG_OUTPUT "Control ENABLE_DEBUG_OUTPUT in core" OFF)
 
 set(VIRTCARD ON CACHE BOOL "Enable VIRTCARD in core" FORCE)
+set(ENABLE_DUMB_DONGLE ON CACHE BOOL "Skip user-presence tests in core" FORCE)
 if(NOT QEMU_DEBUG_OUTPUT)
     set(ENABLE_DEBUG_OUTPUT OFF CACHE BOOL "Disable ENABLE_DEBUG_OUTPUT in core" FORCE)
 endif()

--- a/Src/qemu.c
+++ b/Src/qemu.c
@@ -26,6 +26,7 @@ static void* canokey_emu_state;
 
 // mock device functions
 USBD_StatusTypeDef USBD_LL_Init(USBD_HandleTypeDef *pdev) { return USBD_OK; }
+void USBD_LL_Init_Done(void) {}
 USBD_StatusTypeDef USBD_LL_DeInit(USBD_HandleTypeDef *pdev) { return USBD_OK; }
 USBD_StatusTypeDef USBD_LL_Start(USBD_HandleTypeDef *pdev) {
   USBD_LL_SetSpeed(&usb_device, USBD_SPEED_FULL);

